tests/def_list.md

```diff
@@ -10,13 +10,80 @@ Second Term
 which is multiline
 
 : This is not a correct definition list
+
+# With multiple paragraphs
+
+First Term
+: This is the definition of the first term.
+
+Second Term
+: This is one paragraph of the second term.
+
+  This is another paragraph of the second term.
+
```

Comment:
Perhaps we should include cases for

- mixing lazy multiline and the multi-paragraph markup
- mixing using multiple definition terms with the multi-paragraph markup
- some case like

   ```
   # foo
   : not a definition
   ```